server/index.js

```diff
@@ -467,7 +467,11 @@ export default class Server {
     }
 
     if (hash !== this.buildStats[filename].hash) {
-      throw new Error(`Invalid Build File Hash(${hash}) for chunk: ${filename}`)
```

Comment:
Here's console.log the ^^ message and do return this.send404(res).
In the dev mode, there's no buildId and it's been handled in few lines before.